include/groonga/groonga.h

```diff
@@ -1422,6 +1422,22 @@ GRN_API grn_rc
 grn_obj_flush_recursive_dependent(grn_ctx *ctx, grn_obj *obj);
 GRN_API grn_rc
 grn_obj_flush_only_opened(grn_ctx *ctx, grn_obj *obj);
+/**
+ * \brief Defragment an object to reduce fragmentation in the database file.
+ *
+ * The defragmentation process aims to reduce the fragmentation in the database
+ * file area occupied by the object, based on the provided threshold value.
+ *
+ * \param ctx The context object.
+ * \param obj The target object to defragment.
+ * \param threshold The threshold value used to guide the defragmentation
+ *                  process.
+ *
+ * \return The total number of segments on which defragmentation was performed
+ *         on success.
```

Comment:
途中でエラーが発生したときもその時点までで処理したセグメント数を返すので`on success`はいらなそう。

```suggestion
 * \return The total number of segments on which defragmentation was performed.
```

Reply:
fix: 3a88f46 確かにです！